src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int DEFAULT_RADIUS = 10;
+    public static final String DEFAULT_COLOR = "WHITE";
+
+    public Figure getRandomFigure() {
+        Figure[] figures = {new Square(), new Rectangle(), new RightTriangle(),
+                new Circle(), new IsoscelesTrapezoid()};
+        /*int index = new Random().nextInt(10000);
+        while (index > figures.length - 1) {
+            int divider = new Random().nextInt(100);
+            index /= (divider != 0) ? divider : 1;
+        }*/
+        return figures[new Random().nextInt(figures.length - 1)];
```

Comment:
The expression `new Random().nextInt(figures.length - 1)` will never select the last figure in the array. Use `new Random().nextInt(figures.length)` to ensure all figures can be selected.